mat64/solve.go

```diff
@@ -11,15 +11,15 @@ import (
 	"github.com/gonum/matrix"
 )
 
-// Solve solves a minimum-norm solution to a system of linear equations defined
-// by the matrices a and b. If a is singular or near-singular a Condition error
+// Solve finds a minimum-norm solution to a system of linear equations defined
+// by the matrices A and b. If A is singular or near-singular a Condition error
```

Comment:
a when you're writing about the variable label and A when you are writing about the mathematical entity.